cvat/apps/iam/rules/limitations.rego

```diff
@@ -0,0 +1,34 @@
+package limitations
```

Comment:
Let's rename the file into limits.rego

Reply:
OK. I see that we already have limits. It is an indication, that something wrong.